raiden-cli/src/routes/tokens.ts

```diff
@@ -1,20 +1,52 @@
 import { Router, Request, Response } from 'express';
+import { first } from 'rxjs/operators';
+
 import { Cli } from '../types';
+import { validateAddressParameter } from '../utils/validation';
+
+async function getAllTokens(this: Cli, _request: Request, response: Response) {
+  response.json(await this.raiden.getTokenList());
+}
+
+async function getTokenNetwork(this: Cli, request: Request, response: Response) {
+  // Attention: this also enables monitoring token
+  try {
+    response.json(await this.raiden.monitorToken(request.params.tokenAddress));
+  } catch (error) {
+    response.status(404).send(error.message);
+  }
+}
+
+async function getTokenPartners(this: Cli, request: Request, response: Response) {
+  const token: string = request.params.tokenAddress;
+  const channelsDict = await this.raiden.channels$.pipe(first()).toPromise();
+  const baseUrl = request.baseUrl.replace(/\/\w+$/, '');
+  if (!(token in channelsDict)) response.status(404).send('Unknown token or no partner');
```

Comment:
Having no partner in a network results into an empty list not an error. Please separate these two cases.